db/models.py

```diff
@@ -40,8 +48,95 @@ def __str__(self) -> str:
 
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
-    cinema_hall = models.ForeignKey(to=CinemaHall, on_delete=models.CASCADE)
-    movie = models.ForeignKey(to=Movie, on_delete=models.CASCADE)
+    cinema_hall = models.ForeignKey(
+        to="CinemaHall", on_delete=models.CASCADE, related_name="cinema_halls"
+    )
+    movie = models.ForeignKey(to="Movie", on_delete=models.CASCADE)
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(to="User", on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        to="MovieSession",
+        on_delete=models.CASCADE,
+        related_name="movie_sessions",
+    )
+    order = models.ForeignKey(
+        to="Order", on_delete=models.CASCADE, related_name="orders"
```

Comment:
The related name represents the name of the attribute of the related model through which you can access related `Tickets`. Therefore, it should be `tickets`, not the other way around.